interface/src/app/genesets/use.js

```diff
@@ -12,354 +12,409 @@
  * The dependencies block here is also where component dependencies should be
  * specified, as shown below.
  */
-angular.module( 'tribe.genesets.use', [
```

Comment:
Did you change anything else except the coding style in this file?

Reply:
Never mind, I meant the other parts except `showVersion()`.

Reply:
Nope, that's it 👍